lib/Notes/NoteCreatePage/NoteCreatePage.js

```diff
@@ -14,6 +14,7 @@ import { Icon } from '@folio/stripes-components';
 import NoteForm from '../components/NoteForm';
 import { noteTypesCollectionShape } from '../response-shapes';
 import { referredEntityDataShape } from '../components/NoteForm/noteShapes';
+import { SESSION_STORAGE_KEY_PREFIX } from '../constants';
```

Comment:
For future work, [see the `useNamespace` hook / `withNamespace` HOC](https://github.com/folio-org/stripes-core/blob/0fcafe70b8a25525ef488ab6e7e800c148799e4d/src/components/Namespace/withNamespace.js) in stripes-core.

Reply:
Thanks, I didn't know about this